simulator.py

```diff
@@ -108,41 +108,83 @@ def members():
     def log_encounter(human1, human2, location, duration, distance, time):
 
         human1.events.append(
-            {
-                'human_id': human1.name,
-                'time': time,
-                'event_type': Event.encounter,
-                'payload': {
-                    'encounter_human_id': human2.name,
-                    'other_human_is_infected': human2.is_sick,
-                    'other_human_symptoms': human2.symptoms,
-                    'other_human_test_results': human2.test_results,
+            {   'encounter': {
+                    'time': time,
```

Comment:
Not sure if this change of schema was requested.
But I would keep `time`, `event_type` and `human_id` as a top-level concept: It should always be available for an event generated by a mobile app. But I like the nesting of `encounter` and `human1` and `human2`

Reply:
Thanks for mentioning this, and sorry for making changes without consultation.

Many additions to the payload were requested, and once I added all of them I found it kind of overwhelming and wanted some more structure. I didn't realize that there were requirements from the app side of things; very good to know and I will respect those in future.

In any case this will change because the ML team would like these divided into "observed" and "unobserved". Maybe the schema could be:
{time, 
 event type, 
 human_id, 
 observed { 
    encounter { ....},
    human1 {...},
    human2 {...}
 },
unobserved { 
    encounter { ....},
    human1 {...},
    human2 {...}
 }
}

Does this sound good? I'll also post to channel so this doesn't get lost.